### DISCUSSION TITLE
Fix finding lines in multi-lines secret

### DISCUSSION THREAD

**Discussion by ShiraCohen33** [aquasecurity/trivy#4372] [Q&A]
## Description

In case secret is defined in multiline for example: 
```
secretKey:

   "Aa123456"
```
Trivy saves a finding with the start and end lines of the secret itself without its key and the match value is also the secret without its key.

## What did you expect to happen? 
Save the start and end lines that include the key and the secret to fixing this issue.

## What happened instead?
Save the start and end lines of the secret itself.

## Additional details (base image name, container registry info...):
We run into a problem when we create a suppression rule on Aqua platform that refers to the secret key, and we can't catch it in this case because we don't have the key.

**Comment by knqyf263** [comment]
What do you mean "key"? Do you mean `secretKey` in your example? It isn't designed to do that. How can we determine the key?
